docker/install-dynamic-plugins.py

```diff
@@ -9,16 +9,16 @@ class InstallException(Exception):
     """Exception class from which every exception in this library will derive."""
     pass
 
-def merge(source, destination):
+def merge(source, destination, prefix = ''):
```

Comment:
Can you add a standard copyright header and small usage explanation for this script so others know what it's for, how it's used, etc.  ?